src/Skills.jsx

```diff
@@ -128,23 +126,23 @@ class Skills extends Component {
               <Select
                 name="skills"
                 labelKey="name"
-                value={this.state.skill_type}
                 onChange={onSkillChange}
+                value={this.state.currentSkill.skill_type}
                 options={this.state.skills}
               />
               </form>
             </Col>
-            <Button name="AddSkill"
-                    bsStyle="primary"
-                    onClick={this.openSkillModal}>
-              Add Skill
-            </Button>
+            <Button
+              name="AddSkill"
+              bsStyle="primary"
+              onClick={this.openSkillModal}
+              children="Add Skill" />
```

Comment:
I did it for "readability' but I can change it back. I prefer to have self-closing elements when possible.